Content.Server/IoC/ServerContentIoC.cs

```diff
@@ -57,6 +58,7 @@ public static void Register()
             IoCManager.Register<IAdminLogManager, AdminLogManager>();
             IoCManager.Register<PlayTimeTrackingManager>();
             IoCManager.Register<UserDbDataManager>();
+            IoCManager.Register<MutationSystem>();
```

Comment:
this is not necessary, entity systems are registered normally

Reply:
i imagine this was done to have it be a dep in `PlantHolderComponent`--in that case, just call `_entMan.System<MutationSystem>` where necessary instead

Reply:
Fixed, thanks.